src/components/Votes.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import SendVotes from "../utils/SendVote";
+
+const Votes = (props) => {
+    const [hasVoted,SetVote] = useState(false);
```

Comment:
try to make sure the state and set state have the same name with the setter only having the word 'set' before it - e.g. this should be voted, setVoted and clicked, setClicked